custom/ilsgateway/api.py

```diff
@@ -230,8 +233,11 @@ def location_sync(self, ilsgateway_location, fetch_groups=False):
                 location.longitude = float(ilsgateway_location.longitude)
             location.location_type = ilsgateway_location.type
             location.site_code = ilsgateway_location.code
+            # todo: unicode?
             location.external_id = str(ilsgateway_location.id)
             location.save()
+
+            # todo: shouldn't this only be creating supply points for objects just at the facility level?
```

Comment:
@czue There are some sms users in ILS that are assigned to non-facility locations. In HQ when we assign location to user supply point is automatically created. That's reason why I'm creating supply point for all locations. Not sure how it should be solved.